Patch-Source: https://github.com/llvm/llvm-project/pull/140176
---
From a13945bc81b9f7feb596f09e4f31c8dd04bb9da3 Mon Sep 17 00:00:00 2001
From: Douglas Yung <douglas.yung@sony.com>
Date: Wed, 7 May 2025 06:13:07 +0000
Subject: [PATCH] Fix test pfalse-v4i1.ll added in #138712 to require asserts.

Should fix build bot failure: https://lab.llvm.org/buildbot/#/builders/202/builds/1102

(cherry picked from commit 194a4a333a95f9e001d2c8abe82c3d4cf8894acf)
---
 llvm/test/CodeGen/Hexagon/isel/pfalse-v4i1.ll | 1 +
 1 file changed, 1 insertion(+)

diff --git a/llvm/test/CodeGen/Hexagon/isel/pfalse-v4i1.ll b/llvm/test/CodeGen/Hexagon/isel/pfalse-v4i1.ll
index c0904b8b4fdd6..2c26bb1e310ea 100644
--- a/llvm/test/CodeGen/Hexagon/isel/pfalse-v4i1.ll
+++ b/llvm/test/CodeGen/Hexagon/isel/pfalse-v4i1.ll
@@ -1,4 +1,5 @@
 ; RUN: llc -march=hexagon -debug-only=isel 2>&1 < %s - | FileCheck %s
+; REQUIRES: asserts
 
 ; CHECK: [[R0:%[0-9]+]]:intregs = A2_tfrsi 0
 ; CHECK-NEXT: predregs = C2_tfrrp killed [[R0]]:intregs
